src/mbgl/renderer/renderer_impl.hpp

```diff
@@ -105,6 +105,9 @@ class Renderer::Impl : public GlyphManagerObserver,
     std::unordered_map<std::string, std::unique_ptr<RenderLayer>> renderLayers;
     RenderLight renderLight;
 
+    std::unique_ptr<CrossTileSymbolIndex> crossTileSymbolIndex;
```

Comment:
`CrossTileSymbolIndex` definitely doesn't need to be, I'll change that.

@ansis can say better, but I think the idea with using `unique_ptr<Placement>` is that we can have one `Placement` "in progress" while the previous one is still being used, and then efficiently swap out the new one when it finishes. That's the approach the `start-collision-async` branch uses. I think we're interested in continuing to experiment with the asynchronous approach, so it might be nice to leave this in for now?